.jekyll-cache/Jekyll/Cache/Jekyll--Converters--Markdown/0c/a6b4c9b61c2693a88e1590403c37a62edaac05a8697bb90f8df7195e5b3810

```diff
@@ -0,0 +1,34 @@
+I"�
+<h2 id="project-description">Project Description</h2>
```

Comment:
this file looks pretty funky to me. Are you using an add-on to convert markdown to html? Not sure if it matters